receiver/kafkareceiver/kafka_receiver_test.go

```diff
@@ -708,82 +708,136 @@ func TestNewMetricsReceiver(t *testing.T) {
 
 func TestComponentStatus(t *testing.T) {
 	t.Parallel()
```

Comment:
Since we're using `runTestForClients` which changes the feature gate, I don't think this can run in parallel anymore?